spec/support/authentication.rb

```diff
@@ -0,0 +1,8 @@
+module AuthenticationForFeatureRequest
+  def login user, password = 'login'
```

Comment:
Use def with parentheses when there are parameters.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.